src/main/java/com/radixdlt/crypto/hdwallet/HDWallet.java

```diff
@@ -0,0 +1,27 @@
+/*
+ *
+ *  * (C) Copyright 2020 Radix DLT Ltd
+ *  *
+ *  * Radix DLT Ltd licenses this file to you under the Apache License,
+ *  * Version 2.0 (the "License"); you may not use this file except in
+ *  * compliance with the License.  You may obtain a copy of the
+ *  * License at
+ *  *
+ *  *  http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing,
+ *  * software distributed under the License is distributed on an
+ *  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ *  * either express or implied.  See the License for the specific
+ *  * language governing permissions and limitations under the License.
+ *
+ */
+
+package com.radixdlt.crypto.hdwallet;
+
+import com.radixdlt.crypto.ECKeyPair;
+
+public interface HDWallet {
+	HDKeyPair deriveKeyAtPath(String path);
```

Comment:
Comments required.

Reply:
Yes please ignore documentation for now, will document everything once APIs becomes more stable :)

Reply:
OK.